src/net/sourceforge/kolmafia/request/CharPaneRequest.java

```diff
@@ -1764,51 +1742,48 @@ public static final void parseStatus(final JSONObject json) throws JSONException
 
     CharPaneRequest.setInteraction();
 
-    if (KoLCharacter.getLimitMode().limitFamiliars()) {
-      // No familiar
```

Comment:
honestly I prefer this blank if, skip the rest of the chain to the not + indent.

Reply:
like
```java
if (X) {
  // not relevant because M
} else if (Y) {
  pseudoDoThing();
} else {
  pseudoDoOtherThing();
}
```
is more readable and cleaner than
```java
// not relevant because M
if (!X) {
  if (Y) {
    pseudoDoThing();
  } else {
    pseudoDoOtherThing();
  }
}
```
imo